### ISSUE TITLE
Typo in the "Archive" links? 

### ISSUE THREAD

**Issue by mreferre** [tblyler/light-hugo#6]
Hey there! I am trying to play around with this theme as I am considering it for my migration from WP to a Hugo-bsed blog hosted on S3. It's all great except one tiny thing I am trying to determine if it's a bug or user-error. 

If I deploy the blog + this theme.... the `Archive` hyperlink in both the `footer.html` and `header.html` files point to `/post/`. This gives me a `404`. If I change manually the link in the browser to `<myFQDN>/posts/` it works just fine. 

Is it possible that you have a typo in those two files? Worst case I am planning on using a `sed` in my build instructions to `s*/post/*/posts/` in those two files (I don't want to fork your repo) but it just feels wrong. If you feel this is a typo I am happy to submit a PR for this but I am checking if I am not missing something first. 

Thanks!

**Comment by tblyler** [comment]
Yeah, it appears that the theme is looking for a `content/post` folder rather than a `content/posts` folder. As a temporary fix, you could use a `content/post` folder. I'll check out supporting both or just forcing the use of a `content/posts` folder since that is the typical use case with Hugo.

**Comment by mreferre** [comment]
Thanks. I did not do anything special with Hugo. I thought it was just a typo? Why would you need to support both out of curiosity? My understanding (which is very basic) is that Hugo just use/create the `posts` folder. No? Thanks!
